features/step_definitions/property_address.rb

```diff
@@ -0,0 +1,21 @@
+Given(/^I do not have a property address to provide$/) do
+  @deed = Deed.new(1)
+  @deed.property_address = ''
+  @deed_hash = @deed.to_hash
+end
+
+Given(/^I have a property address that is formatted with commas$/) do
+  @deed = Deed.new(1)
+  @deed.property_address = '256 King Henrys Drive, New Addington, Croydon, '\
+  'CR7 6TH'
+  @deed.borrowers[0][:dob] = '21/09/1971'
```

Comment:
Just a question, why does the date of birth need to be set to something that is not the default?